Fix error when deleting a filter tag applied in the FilterBar component (#1005)
vtex/styleguide: CHANGELOG.md

## [Unreleased]

### Fixed

> - Fix an error that can be caused when deleting a filter tag applied in the FilterBar component

Reviewer: ```suggestion
- Error caused by deleting a filter tag while the `Menu` is still active in the `FilterBar` component.
```
Maybe?

Author: Yes, i think it's better this way
